HTTPServer/HttpProcessor.cs

```diff
@@ -185,6 +187,34 @@ public void HandleClient(TcpClient tcpClient)
                                                 else
                                                     response = HttpResponse.Send(res, "text/xml");
                                             }
+                                            else if ((Host == "away.veemee.com" || Host == "home.veemee.com") && absolutepath.EndsWith(".php"))
+                                            {
+                                                LoggerAccessor.LogInfo($"[HTTP] - {clientip}:{clientport} Requested a VEEMEE  method : {absolutepath}");
+
+                                                VEEMEEClass veemee = new(request.Method, absolutepath); 
+                                                if (request.GetDataStream != null)
+                                                {
+                                                    using MemoryStream postdata = new();
+                                                    request.GetDataStream.CopyTo(postdata);
+
+                                                    postdata.Position = 0;
+                                                    // Find the number of bytes in the stream
+                                                    int contentLength = (int)postdata.Length;
+                                                    // Create a byte array
+                                                    byte[] buffer = new byte[contentLength];
+                                                    // Read the contents of the memory stream into the byte array
+                                                    postdata.Read(buffer, 0, contentLength);
+                                                    var res = veemee.ProcessRequest(buffer, request.GetContentType(), absolutepath);
+                                                    postdata.Flush();
+
+                                                    veemee.Dispose();
+
+                                                    if (string.IsNullOrEmpty(res.Item1))
+                                                        response = HttpBuilder.InternalServerError();
+                                                    else
+                                                        response = HttpResponse.Send(res.Item1, "text/xml");
```

Comment:
This line needs an adjustment.

The tuple data was designed so the content type can be passed in.

Veemee can send booth json and xml data, so this needs to be res.Item2 instead of text/xml.

Reply:
Yeah I noticed the implementation of the tuple when I was reworking it, and was using the res.Item1 as reference from HTTPSServer, wouldn't Items2 be sending the content-type instead and not the data that need to be sent?

Reply:
Oh i see what you mean by putting res.Item2 as content-type now, updating this.
![image](https://github.com/GitHubProUser67/MultiServer2/assets/26616362/428b615a-8b4f-43f9-9c24-e8a367790722)